### ISSUE TITLE
"Move to trash" does not do anything (cross-filesystem)

### ISSUE THREAD

**Issue by Yenya** [BestImageViewer/geeqie#1025]
### ISSUE TYPE
* Bug Report

### GEEQIE VERSION
```
Geeqie 1.7.3 GTK3

```

### OS / DISTRIBUTION
Fedora 36, x86_64, XFCE desktop

### SUMMARY
When I either hit the Delete key or select "Move to trash" from the right-click menu, the confirmation dialog pops up. But clicking on its Delete button does not delete the image, it stays selected and displayed as a current image instead. No further error message is displayed, neither on STDERR nor in the X dialog.

Using Shift-Delete (or selecting "Delete..." from the right-click menu) works as expected.

I tried to run "strace -f -p <main_PID_of_geeqie>", and I don't see a rename() or unlink() call on the original image (only on a thumbnail file in ~/.cache), or any suspicious ENOENT/EXDEV return values (there are lots of ENOENTs for icons and stuff, of course).

The image directory in question sits on a different volume than my home directory (which presumably contains the .Trash directory), which is why I mention EXDEV here.

When I try to do the same in a subdirectory which does reside on the same filesystem as my $HOME, moving to trash apparently works.

**Comment by caclark** [comment]
Is #669 related to this?

**Comment by Yenya** [comment]
Yes, it definitely looks similar. But in my case, there is not a private mount, just a global mount of another filesystem, where I keep large data such as photos.
